lib/content-services/src/lib/agent/services/agent.service.spec.ts

```diff
@@ -15,49 +15,83 @@
  * limitations under the License.
  */
 
-import { AgentService } from './agent.service';
 import { TestBed } from '@angular/core/testing';
-import { AgentPaging } from '@alfresco/js-api';
-import { ContentTestingModule } from '../../testing/content.testing.module';
+import { AlfrescoApiService, CoreTestingModule } from '@alfresco/adf-core';
+import { AgentPaging, AgentsApi, AgentWithAvatar } from '@alfresco/js-api';
+import { AgentService } from '@alfresco/adf-content-services';
+
+const agent1 = {
+    id: '1',
+    name: 'HR Agent',
+    description: 'Your Claims Doc Agent streamlines the extraction, analysis, and management of data from insurance claims documents.'
+};
+
+const agent2 = {
+    id: '2',
+    name: 'Policy Agent',
+    description: 'Your Claims Doc Agent streamlines the extraction, analysis, and management of data from insurance claims documents.'
+};
+
+const agentPagingObjectMock: AgentPaging = {
+    list: {
+        entries: [
+            {
+                entry: agent1
+            },
+            {
+                entry: agent2
+            }
+        ]
+    }
+};
+
+const avatarAgentMock: string = 'https://res.cloudinary.com/hyld/image/upload/f_auto,c_fill,g_auto,w_1400,h_730/v1/h2/hero/blue-shirt-woman';
```

Comment:
no need to set type, it's inferred automatically based on value (string) which you assign